products/goland.md

```diff
@@ -0,0 +1,44 @@
+---
+title: GoLand Plugin Development
+---
+
+## Introduction
+GoLand is an IntelliJ Platform-based product.
+Plugin projects for GoLand can be developed using IntelliJ IDEA with the `gradle-intellij-plugin`.
+
+## Configuring Plugin Projects Targeting GoLand
+The configuration of GoLand plugin projects follows the methods described in [Configuring Plugin Projects using the IntelliJ IDEA Product Attribute](dev_alternate_products.md#configuring-plugin-projects-using-the-intellij-idea-product-attribute), and [Configuring the plugin.xml File](dev_alternate_products.md#configuring-pluginxml).
+
+The table below summarizes the `gradle-intellij-plugin` attributes to set in the `build.gradle` file for a GoLand plugin project:
+
+| `gradle-intellij-plugin` Attribute | <br>Attribute Value |
+|-----------|-------|
+| [`intellij.type`](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#intellij-platform-properties) | `IU` for IntelliJ IDEA Ultimate.  |
+| [`intellij.version`](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#intellij-platform-properties) | Set to the same `IU` BRANCH.BUILD as the GoLand target version, e.g. `192.7142.36` |
+| [`intellij.plugins`](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#intellij-platform-properties) | `org.jetbrains.plugins.go:192.6603.23.335` for the Go plugin.<br>See below for Go plugin version information. |
+| [`runIde.ideaDirectory`](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#running-dsl) | Path to locally installed target version of GoLand. For example, on macOS:<br>`/Users/<user name>/Library/Application Support/JetBrains/Toolbox/apps/Goland/ch-0/192.7142.48/GoLand.app/Contents` |
+
+The required `org.jetbrains.plugins.go` plugin isn't compatible with IntelliJ IDEA Community edition but is compatible with IntelliJ IDEA Ultimate (`IU`) edition.
+Product compatibility is determined from the Go plugin [version page](http://plugins.jetbrains.com/plugin/9568-go/versions). 
+The Go plugin isn't bundled with `IU`, so the Go plugin version must be explicitly declared and support the target GoLand (and `IU`) BRANCH.BUILD version. 
```

Comment:
AFAIU "take Go plugin version matching IJ release version"? then we could shorten this paragraph

Reply:
1. The GoLand target BRANCH.BUILD version determines the IU BRANCH.BUILD version.
2. The IU BRANCH.BUILD version determines the version of the Go plugin.

I don't know of a way to determine the Go plugin version directly from the GoLand BRANCH.BUILD version. In any case, the reader would still have to determine the compatible IU version.

Or am I missing your point?

Reply:
my understanding is, simply take version number for Goland plugin that matches exactly version number for IDE you're targeting